api/config/appstudio.redhat.com_enterprisecontractpolicies.yaml

```diff
@@ -189,7 +189,7 @@ spec:
                                   format: date-time
                                   type: string
                                 imageRef:
-                                  description: ImageRef is used to specify an image by its digest.
+                                  description: ImageDigest is used to specify an image by its digest.
```

Comment:
Is there supposed to be an imageDigest key in this file?

Reply:
Yes! Good catch, I never updated the json tag. Done now.